Makefile

```diff
@@ -360,17 +360,14 @@ generate-dockerfiles: install-tool-generate
 # Generates all files that can be generated, includes release files, code generation
 # and updates vendoring.
 # Use CURRENT_VERSION_IMAGES="<branch>" if you need to override the defaulting to main
-generated-files: update-tekton-pipelines install-tools generate-dockerfiles release-files
```

Comment:
just to be sure: this is removed, as we let the konflux-gen automation in hack handle it?